index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>サイト制作課題</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Anton&family=Noto+Sans+JP:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header class="header">
+      <h1 class="header-logo-wrapper">
+        <img src="./img/logo.svg" alt="ロゴ" class="header-logo" />
```

Comment:
```suggestion
        <img src="./img/logo.svg" alt="Grand Players" class="header-logo" />
```
`alt` 属性は、目の見えない人がスクリーンリーダーを使ってWEBサイトを閲覧する際に読み上げられる文言であったり、何らかの不具合で画像が表示できなかった場合の代替テキストを定義します。
今回のロゴのように、画像が表現している内容が文字の場合は、その文字を`alt`属性に入れるのが適切です。

[教材のimgに関する部分](https://www.notion.so/shinonome-inc/img-4cb5ecb9ce9d492396e4249afbf63183?pvs=4)でもaltについて軽く触れているので、参考にしてみてください。